framework/tests/value/ValueCast.java

```diff
@@ -0,0 +1,11 @@
+// Test case for issue 1264: https://github.com/typetools/checker-framework/issues/1264
```

Comment:
#1264 is about casts, but this test case doesn't have a cast in it.  I think it will not be fixed by the fix for that issue.